### PR TITLE
tests: for inventory and icons in shared status folder

### PR DESCRIPTION
**Pull request by debjyoti-pandit**

### CODE REVIEW

File: packages/shared/status/Inventory.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+jest.mock('./Node', () => ({
+  nodeStatus: (resource) => resource.status,
+}));
+
+describe('getStatusGroups', () => {
+  const resources = [
+    { id: 1, status: 'Not Ready' },
+    { id: 2, status: 'Ready' },
+    { id: 3, status: 'UNKNOWN' },
+    { id: 4, status: 'Not Ready' },
```
Comment (bipuladh) [inline packages/shared/status/Inventory.spec.tsx:18]: Don't we have `enums` for these Statuses that we can use here?

File: packages/shared/status/Inventory.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  getPVStatusGroups,
+  getStatusGroups,
+  InventoryStatusGroup,
+} from './Inventory';
+
+jest.mock('./Node', () => ({
+  nodeStatus: (resource) => resource.status,
+}));
+
+describe('getStatusGroups', () => {
```
Comment (bipuladh) [inline packages/shared/status/Inventory.spec.tsx:13]: Please write better descriptions

File: packages/shared/status/Inventory.spec.tsx (hunk 1 above)
Comment (debjyoti-pandit) [reply packages/shared/status/Inventory.spec.tsx:18]: We have enums, but these are the test values that will get mapped to the proper enums which we have.

Also, ready and not ready like the below pics written in text getting mapped to the enums which are present.

Pic 1 - 
image

Pic 2 -
image

**Pull request merged**